pkg/intermediate/aggregate.go

```diff
@@ -366,6 +366,16 @@ func (a *AggregationProcess) addOrUpdateRecordInMap(flowKey *FlowKey, record ent
 				}
 			}
 		} else {
+			// For inter-node traffic with deny np or drop anp, we may receive the record from the conntrack table first.
+			// In this case, the record from the conntrack table will be considered to do correlation job and the ReadyToSend
+			// is false until we receive another record from PacketIn from another node. However, the record From PacketIN
+			// is not correlationRequired, so we won't send this kind of connection due to this conflict.
+			aggregationRecord.ReadyToSend = true
```

Comment:
I see similar logic later in the function, can we unify?

Reply:
Yes. I've moved it to the bottom and set ReadyToSend and areCorrelatedFieldsFilled to true if the flow is not correlationRequired.